packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5133,6 +5133,981 @@ curl -v -X GET \
+-d '{
+    "client_secret" : "3vimrC5Yv6bSDJzrUdLEYvkf9ElwUeWdndO5nhYp"
+}' "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/secrets/"
+```
+
+##### Response example
+
+```json
+{
+  "id": "ocsap6nz61rKdsoyOY0h7",
+  "client_secret": "3vimrC5Yv6bSDJzrUdLEYvkf9ElwUeWdndO5nhYp",
+  "secret_hash": "0xbeefbeefbeefbeefbeefbeef",
+  "created": "2022-01-01T04:22:27.000Z",
+  "status": "ACTIVE",
+  "_links" : {
+    "deactivate": {
+      "href": "https://${yourOktaDomain}/api/v1/apps/0oad5lTSBOMUBOBVVQSC/credentials/secrets/ocsap6nz61rKdsoyOY0h7/lifecycle/deactivate",
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    }
+  }
+}
+```
+
+If `client_secret` in the request is more than 100 characters long, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: client_secret",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "client_secret: 'client_secret' cannot be more than '100' characters long."
+    }
+  ]
+}
+```
+
+If `client_secret` in the request is less than 14 characters long, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: client_secret",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "client_secret: The field is too short"
+    }
+  ]
+}
+```
+
+If `client_secret` in the request is less than 32 characters long, and the `token_endpoint_auth_method` of the app is `client_secret_jwt`, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: client_secret",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "client_secret: 'client_secret' must be at least '32' characters long for client_secret_jwt auth method."
+    }
+  ]
+}
+```
+
+If `token_endpoint_auth_method` for the app is `private_key_jwt`, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: OAuth2ClientSecretMediated",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "'client_secret' cannot be used when 'token_endpoint_auth_method' is 'private_key_jwt'."
+    }
+  ]
+}
+```
+
+If `client_secret` in the request contains non ASCII characters, you receive an error response.
+
+```json
+{
+  "errorCode": "E0000001",
+  "errorSummary": "Api validation failed: client_secret",
+  "errorLink": "E0000001",
+  "errorId": "oaeMHrsk2WLTACvPU5T7yQ4yw",
+  "errorCauses": [
+    {
+      "errorSummary": "client_secret: ''client_secret'' must only contain printable ASCII: [x20-x7E]+"
+    }
+  ]
+}
+```
+
+If the number of client secrets per application exceeds the set limit (currently 2), you receive an error response.
```

Comment:
When it says `(currently 2)` it indicates that this might change. But we would need to update the doc if that changes as we mention the number 2 here, so we might as well just say:
```suggestion
If the number of client secrets per application exceeds 2, you receive an error response.
```

Reply:
updated